core/store/src/trie/mod.rs

```diff
@@ -115,6 +115,14 @@ impl TrieNodeWithSize {
     }
 }
 
+#[derive(Debug, Clone)]
+pub enum Level {
```

Comment:
This is just a placeholder for a type. Will probably change it; open to good ideas :)